src/docker-utils.coffee

```diff
@@ -101,27 +101,30 @@ do ->
 		Promise.using readLockImages(), ->
 			dockerProgress.pull(image, onProgress)
 
-	supervisorTag = config.supervisorImage
```

Comment:
Well it definitely needs to be error checked, but having something synchronous wrapped in a promise does mean you have to deal with async stuff when you don't necessarily have to